graphql-cli/src/test/resources/expectedGenCode/functionSignature/initFunctionSignatureWithApiKeysConfig.bal

```diff
@@ -1 +1 @@
-(ApiKeysConfig apiKeysConfig, string serviceUrl, http:ClientConfiguration clientConfig = {}) returns graphql:ClientError?
+(ApiKeysConfig apiKeysConfig, string serviceUrl, ConnectionConfig config) returns graphql:ClientError?
```

Comment:
Shouldn't we provide the default `{}` value for the `ConnectionConfig config`

Reply:
Updated in https://github.com/ballerina-platform/graphql-tools/commit/1af52a4706e5087de8621c9e7ce382bbf445b16e